build/packaging/srpm/condor.spec

```
@@ -572,6 +572,11 @@ Group: Applications/System
 Requires: %name = %version-%release
 Requires: python3-condor = %{version}-%{release}
 Requires: python3-six
+%if 0%{?rhel} == 7
```

Let make it work for RHEL 7, not Amazon Linux 2, and RHEL > 8.
```suggestion
if 0%{?rhel} == 7 && ! 0%{?amzn}
```